crypto/CMakeLists.txt

```diff
@@ -1,5 +1,15 @@
 # crypto
 
-add_lib_batch(vmecs-crypto STATIC "*.c")
+# Option to select the crypto library
+option(USE_MBEDTLS "Use MbedTLS instead of OpenSSL" ON)
```

Comment:
Better to use USE_ENCLIB with following possible values: `openssl-evp, mbedtls, gnutls` instead